tests/post_training/requirements.txt

```diff
@@ -0,0 +1,11 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+torch
+--extra-index-url https://download.pytorch.org/whl/cpu
+torchvision
+onnx
+onnxruntime
+pytest
+openvino>=2022.1
+openvino-dev>=2022.1
```

Comment:
```suggestion
openvino-dev>=2022.1
```

Reply:
> Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately.

https://docs.openvino.ai/latest/openvino_docs_install_guides_install_dev_tools.html#for-python-developers